languages/elm/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,4 @@
+In this exercise you're going to write some minimal Elm code, to get a feel for the basics
+
+You should implement the `greet` function, to return "Welcome ", plus the `personToGreet` in upper case.
+
```

Comment:
This document does not entirely conform to [the specs])(https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsinstructionsmd). For example, there is not really a story/theme here. Additionally, there should be individual tasks for each task the student should be doing (see [this example](https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/lucians-luscious-lasagna/.docs/instructions.md)). Those tasks should include an example of calling the function to implement and its return value.

The exercise itself reminds me quite a bit of the existing `two-fer` exercise. Was there any reason you didn't copy any of the existing basics exercises? Like this [F# implementation](https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/lucians-luscious-lasagna/.docs/instructions.md). One major benefit of re-using the existing exercise is that you can also re-use a lot of the existing documentation files. That said, you don't have to re-use an existing exercise of course, it's just that the `basics` concept exercise is a very tricky one to get right. For example, in this exercise the student is expected to work with strings, but the introduction doesn't actually introduce strings. For this reason, the other basics exercises all work with basic integers (no floats), as those will be familiar to anyone and don't require much introduction.

Reply:
The reason strings are used is that it requires you to use an `import`, which is part of what we defined as the basics. The string concatentation is in the hints, and in the test, but it could go in the introduction as well if you like?

`two-fer` in Elm uses a `Maybe` and potentially pattern matching, its actually reasonably far down the line in terms of Elm concepts.

I'm obviously not very keen to rewrite the exercise, but will leave the decision up to you. I think one issue for me is that the specs change faster than I can keep up with ...